app/Http/Controllers/RecommendationController.php

```diff
@@ -9,26 +9,34 @@
 use App\Models\User;
 use App\Models\Email;
 
-
 class RecommendationController extends \Controller
 {
     protected $settings;
 
     protected $rules = [
-    'rank_character'  => 'required',
-    'rank_additional' => 'required',
-    'essay1'          => 'required',
+      'rank_character'  => 'required',
+      'rank_additional' => 'required',
+      'essay1'          => 'required',
+    ];
+
+    protected $applicant_rules = [
```

Comment:
Okay yeah this is probably the most confusing part. So the normal rules get checked [here](https://github.com/DoSomething/longshot/pull/764/files#diff-4f2c714c5841af863b7dbb3b575ff548R186) and last I checked that was working for recommenders. I do think where that gets called is confusing though, so I'll try moving it into the `updateAdmin` function which makes way more sense to me. 

I'll double check that all the recommender stuff validates correctly and use both arrays.